README.md

```diff
@@ -1,42 +1,24 @@
 # Terraform module for Sourcegraph executors (AWS)
 
-This repository provides a [Terraform module](https://learn.hashicorp.com/tutorials/terraform/module-use?in=terraform/modules) to provision Sourcegraph executor compute resources on AWS.
+This repository provides a [Terraform module](https://learn.hashicorp.com/tutorials/terraform/module-use?in=terraform/modules) to provision [Sourcegraph executor](https://docs.sourcegraph.com/admin/executors) compute resources on AWS. If you are installing executors for the first time, [follow our complete setup guide](https://docs.sourcegraph.com/admin/deploy_executors).
 
 ![Infrastructure overview](https://raw.githubusercontent.com/sourcegraph/terraform-aws-executors/master/images/infrastructure.png)
 
-## Usage examples
+This repository provides four submodules:
 
-There are several examples in the [`examples`](./examples) directory.
+1. The [executors module](https://registry.terraform.io/modules/sourcegraph/executors/aws/latest/submodules/executors) provisions compute resources for executors.
+1. The [docker-mirror module](https://registry.terraform.io/modules/sourcegraph/executors/aws/latest/submodules/docker-mirror) provisions a Docker registry pull-through cache.
+1. The [networking module](https://registry.terraform.io/modules/sourcegraph/executors/aws/latest/submodules/networking) provisions a network to be shared by the executor and Docker registry resources.
+1. The [credentials module](https://registry.terraform.io/modules/sourcegraph/executors/aws/latest/submodules/credentials) provisions credentials required by the Sourcegraph instance to enable observability and auto-scaling of executors.
 
-#### [`examples/single-executor`](./examples/single-executor)
+The [multiple-executors example](https://github.com/sourcegraph/terraform-aws-executors/blob/master/examples/multiple-executors) uses the submodule directly to provision multiple executor resource groups performing different types of work. Follow this example if you are:
 
-This example uses the [root module](https://registry.terraform.io/modules/sourcegraph/executors/aws/latest) that provisions a network, a Docker registry mirror, and a set of resources to run _one_ type of executor. To provision more than one type of executor (multiple queues or multiple environments), see the following `multiple-executors` example.
+1. Provisioning executors for use with multiple features (e.g., both [auto-indexing](https://docs.sourcegraph.com/code_intelligence/explanations/auto_indexing) and [server-side batch changes](https://docs.sourcegraph.com/batch_changes/explanations/server_side)), or
+1. Provisioning resources for multiple Sourcegraph instances (e.g., test, prod)
 
-The following variables must be supplied:
+This repository also provides a [root module](https://registry.terraform.io/modules/sourcegraph/executors/aws/latest) combining the executors, network, and docker-mirror resources into an easier to use package.
```

Comment:
Broken infra image on that page:

<img width="820" alt="CleanShot 2021-11-23 at 11 50 18@2x" src="https://user-images.githubusercontent.com/1387653/143085868-010053b6-3bb9-4491-b2f0-3d9b97f0b578.png">